server.js

```diff
@@ -0,0 +1,18 @@
+
+var express = require('express'),
+  port = process.env.PORT || 3000,
+  app = express();
+
+app.use(express.static('./'));
+app.get('projects.html', function(){
+  response.json({message: 'what is happening'});
+});
+
+// app.get('*', function(request, response){
+//   console.log('New request: ', request.url);
+//   response.sendFile('index.html', {root: '.'});
+// });
+
```

Comment:
Uncomment this to get the server running.

Also be sure to add all sections you want to hide/show via the controller functions in the index.html file.